src/commands/manifest.ts

```diff
@@ -9,13 +21,58 @@ export default class Manifest extends Command {
     {name: 'path', description: 'path to plugin', default: '.'},
   ]
 
+  static flags = {
+    jit: Flags.boolean({
+      allowNo: true,
+      summary: 'append commands from JIT plugins in manifest',
+      default: true,
+    }),
+  }
+
   async run(): Promise<void> {
+    const {flags} = await this.parse(Manifest)
     try {
       fs.unlinkSync('oclif.manifest.json')
     } catch {}
 
     const {args} = await this.parse(Manifest)
     const root = path.resolve(args.path)
+
+    const packageJson = fs.readJSONSync('package.json') as { oclif: { jitPlugins: Record<string, string> } }
+
+    let jitPluginManifests: Interfaces.Manifest[] = []
+
+    if (flags.jit && packageJson.oclif.jitPlugins) {
```

Comment:
@mdonnalley I know this is creating the manifest, but it seems that the logic to locate and resolve the correct version might be something useful in core. Thoughts?